openedx/core/djangoapps/cors_csrf/middleware.py

```diff
@@ -63,13 +65,31 @@ class CorsCSRFMiddleware(CsrfViewMiddleware, MiddlewareMixin):
 
     def __init__(self, *args, **kwargs):
         """Disable the middleware if the feature flag is disabled. """
+
+        # .. custom_attribute_name: tmp_cors_csrf.is_activated
+        # .. custom_attribute_description: Boolean flag to know if CorsCSRFMiddleware is activated
+        set_custom_attribute('tmp_cors_csrf.is_activated', settings.FEATURES.get('ENABLE_CORS_HEADERS', False))
```

Comment:
Should we leave off the default and have the exact response that is in the conditional?